pkg/provider/github/github.go

```diff
@@ -600,5 +600,5 @@ func (v *Provider) isBranchContainsCommit(ctx context.Context, runevent *info.Ev
 	if branchInfo.Commit.GetSHA() == runevent.SHA {
 		return nil
 	}
-	return fmt.Errorf("provided branch %s does not contains sha %s", branchName, runevent.SHA)
+	return fmt.Errorf("provided sha %s is not HEAD commit of branch %s", runevent.SHA, branchName)
```

Comment:
```suggestion
return fmt.Errorf("provided SHA %s is not the HEAD commit of the branch %s", runevent.SHA, branchName)
```

The changes:
1. Capitalized "SHA" as it's an acronym
2. Added "the" before "HEAD commit"
3. Added "the" before "branch"